packages/next-app/src/contexts/StableSwapContext.tsx

```diff
@@ -35,6 +41,22 @@ interface StableSwapContextType {
   setBaselineData: (data: AnalysisData) => void;
   handleImportPoolParametersById: (data: PoolAttribute) => void;
   newPoolImportedFlag: boolean;
+  isGraphLoading: boolean;
+  setIsGraphLoading: (value: boolean) => void;
+  preparePoolPairData: ({
+    indexIn,
+    indexOut,
+    swapFee,
+    allBalances,
+    amp,
+  }: {
+    indexIn: number;
+    indexOut: number;
+    swapFee: number;
+    allBalances: number[];
+    amp: number;
+  }) => StablePoolPairData;
+  numberToOldBigNumber: (number: number, decimals?: number) => typeof bnum;
```

Comment:
maybe we could export all the number and big number converts (opposite direction too)